app/lib/pdf_filler/md502_pdf.rb

```diff
@@ -131,6 +143,26 @@ def generate_codes_for_502_su
       applicable_codes
     end
 
+    def full_names_of_bank_account_holders
+      return nil unless @intake.payment_or_deposit_type.to_sym == :direct_deposit
+
+      if @intake.has_joint_account_holder_yes?
+        account_holder_full_name + " and " + account_holder_full_name(for_joint: true)
+      else
+        account_holder_full_name
+      end
```

Comment:
generally we pull information from the XML -- not sure if I should pull from the XML using `xml_document.at("FirstName")` etc instead of directly calling these methods on the intake?